### PR TITLE
Update set-up-sso-saml-2.0.md

### PR DESCRIPTION
**Pull request by BrJan**
Add note on expiring certs

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by BrJan — COMMENTED** [review]
Replace Okta admin with IdP admin

File: website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ To complete setup, follow the steps below in dbt Cloud:
    | Log&nbsp;in&nbsp;with | SAML 2.0 |
    | Identity&nbsp;Provider&nbsp;SSO&nbsp;Url | Paste the **Identity Provider Single Sign-On URL** shown in the IdP setup instructions |
    | Identity&nbsp;Provider&nbsp;Issuer | Paste the **Identity Provider Issuer** shown in the IdP setup instructions |
-   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions |
+   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions; **Note:** When the cerificate expires, an Okta admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access |
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md]:
```suggestion
   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions; **Note:** When the certificate expires, an Okta admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access |
```

**Review by mirnawong1 — APPROVED** [review]
hey @BrJan , looks great! fixed a small typo and it's good to merge!

File: website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md [hunk 2]
```diff
@@ -424,7 +424,7 @@ To complete setup, follow the steps below in dbt Cloud:
    | Log&nbsp;in&nbsp;with | SAML 2.0 |
    | Identity&nbsp;Provider&nbsp;SSO&nbsp;Url | Paste the **Identity Provider Single Sign-On URL** shown in the IdP setup instructions |
    | Identity&nbsp;Provider&nbsp;Issuer | Paste the **Identity Provider Issuer** shown in the IdP setup instructions |
-   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions |
+   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions; **Note:** When the certificate expires, an Okta admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access |
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/set-up-sso-saml-2.0.md]:
```suggestion
   | X.509&nbsp;Certificate | Paste the **X.509 Certificate** shown in the IdP setup instructions; <br />**Note:** When the certificate expires, an Okta admin will have to generate a new one to be pasted into dbt Cloud for uninterrupted application access. |
```

**Pull request merged**